.github/workflows/test.yml

```
@@ -0,0 +1,85 @@
+        run: npm run test:coverage
+        continue-on-error: true
+
+      - name: Upload coverage to artifacts
+        uses: actions/upload-artifact@v3
+        if: always()
+        with:
+          name: coverage-report
+          path: coverage/
+
+      - name: Comment PR with test results
+        uses: actions/github-script@v6
```

actionlint (1.7.8)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)